enterprise/backend/src/metabase_enterprise/advanced_permissions/api/application.clj

```diff
@@ -1,27 +1,27 @@
-(ns metabase-enterprise.advanced-permissions.api.general
-  "`/advanced-permisisons/general` Routes.
-  Implements the Permissions routes needed for general permission - a class of permissions that control access to features
+(ns metabase-enterprise.advanced-permissions.api.application
+  "`/advanced-permisisons/aplication` Routes.
```

Comment:
@qnkhuat Can you do a followup PR, which renames `aplication` to `application` ?